src/main/webapp/scripts/novncclient.js

```diff
@@ -0,0 +1,79 @@
+import RFB from 'https://cdn.jsdelivr.net/npm/@novnc/novnc@1.1.0/core/rfb.js';
+
+/**
+ * The NoVNCClient class essentially surrounds the RFB object of the noVNC
+ * library, taking care of remoting, what happens if someone connects, 
+ * and what happens if someone disconnects. The class also allows for 
+ * additional functionality that are specific to the scope of Virtual Movie
+ * Night, like changing the viewOnly property of the RFB object (or all
+ * owing who is able to controll).
+ */
+class NoVNCClient {
+  /**
+   * Initalizes a NoVNCClient object.
+   * @param {function(): void} connectCallback This function is called 
+   *    once the rfbConnection connects.
+   * @param {function(): void} disconnectCallback This function is called
+   *    once the rfbConnection disconnects.
+   * @param {HTMLElement} rfbConnectionElement A block HTMLElement that 
+   *    specifies where the RFB object should attach itself.
+   */
+  constructor(connectCallback, disconnectCallback, rfbConnectionElement) {
+    /**
+     * Represents the current noVNC RFB object of the 
+     * session, the single connection to the VNC server of the
```

Comment:
nit: this represents both the object and the connection?

Reply:
the noVNC api defines the RFB object as the single connection to a VNC server so I think so

Reply:
Ok cool, can you update the comment to make that clearer?